particles/Tutorial/Javascript/7_RefinementTypes/RefinementTypes.arcs

```diff
@@ -0,0 +1,41 @@
+// Define a schema that allows us to store a person's name 
+schema Person
```

Comment:
As we discussed, remove the tutorials as they don't have any refinement types... yet :)

Reply:
Can we add tutorials but mark them unfinished? I think it'd be good to have them as something to test against.

Maybe putting them in a `particles/experimental` directory would make more sense than putting them here?

Reply:
I think particles/experimental is a good solution.